src/renderer/components/+cluster-settings/components/cluster-proxy-setting.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { Cluster } from "../../../../main/cluster";
+import { Input } from "../../input";
+import { Spinner } from "../../spinner";
+import { ClusterStore } from "../../../../common/cluster-store"
+import { Icon } from "../../icon";
+import { Tooltip, TooltipContent } from "../../tooltip";
+import { autobind } from "../../../utils";
+
+enum ValueStatus {
+  CLEAN = "clean",
+  DIRTY = "dirty",
+  UPDATING = "updating",
+  ERROR = "error",
+  UPDATED = "updated",
+}
```

Comment:
Why it's repeated over and over again in other files?
Maybe it's better to handler directly in `input.tsx`?